apps/templates/content_templates_test.py

```diff
@@ -76,8 +76,7 @@ def test_render_content_template(self):
         template = {
             '_id': 'foo',
             'template_name': 'test',
-            'template_desk': 'sports',
-            'template_stage': 'schedule',
+            'template_desks': ['sports'],
```

Comment:
Sorry @mugurrus my previous comment was not clear. I wanted to say that a new behave test would be useful to cover a scenario where:
- A template with multiple desks created
- A story created from that template
- Assert that story is created on the desired desk.